tests/globalhelper/runtimeclass.go

```diff
@@ -12,7 +12,19 @@ import (
 )
 
 func CreateRunTimeClass(rtc *nodev1.RuntimeClass) error {
-	rtc, err := GetAPIClient().RuntimeClasses().Create(context.Background(), rtc, metav1.CreateOptions{})
+	// Check for existing runtimeclass first
+	rtcCreated, err := isRtcCreated(rtc)
```

Comment:
Does `Eventually` do the same below?

Reply:
Yeah it's a re-use of the same function that the Eventually uses.